common/functions/src/scalars/function.rs

```diff
@@ -55,19 +97,17 @@ pub struct Range {
 #[derive(Clone)]
 pub enum MonotonicityNode {
     // Describe a function monotonicity information for the range of data.
-    Function(Monotonicity, Option<Range>),
-    Constant(DataValue),
-    Variable(String, Option<Range>),
+    Function(Monotonicity, Range),
```

Comment:
Maybe `Function(Monotonicity, Range)` and `Variable(String, Range)` can be combined into one enum.